packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -693,87 +694,92 @@ function getTreeStatsCore(snapshotTree: ISnapshotTree, stats: ITreeStats): void
  * @param epochTracker - epoch tracker used to add/validate epoch in the network call.
  * @returns fetched snapshot.
  */
-export async function downloadSnapshot(
-	odspResolvedUrl: IOdspResolvedUrl,
-	getAuthHeader: InstrumentedStorageTokenFetcher,
-	tokenFetchOptions: TokenFetchOptionsEx,
-	loadingGroupIds: string[] | undefined,
-	snapshotOptions: ISnapshotOptions | undefined,
-	snapshotFormatFetchType?: SnapshotFormatSupportType,
-	controller?: AbortController,
-	epochTracker?: EpochTracker,
-	scenarioName?: string,
-): Promise<ISnapshotRequestAndResponseOptions> {
-	// back-compat: This block to be removed with #8784 when we only consume/consider odsp resolvers that are >= 0.51
-	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
-	const sharingLinkToRedeem = (odspResolvedUrl as any).sharingLinkToRedeem;
-	if (sharingLinkToRedeem) {
-		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-		odspResolvedUrl.shareLinkInfo = { ...odspResolvedUrl.shareLinkInfo, sharingLinkToRedeem };
-	}
+export const downloadSnapshot = mockify(
```

Comment:
These diffs (and the other ones that employ `mockify`) are misleading. Nothing has changed, other than that the function has been converted to an => function and wrapped with `mockify()`.